target/linux/ath79/dts/ar7161_netgear_wndr3700.dts

```diff
@@ -0,0 +1,38 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "ar7161_netgear_wndr3700.dtsi"
+
+/ {
+	compatible = "netgear,wndr3700", "qca,ar7161";
+	model = "Netgear WNDR3700";
+};
+
+&partitions {
+	partition@0 {
+		label = "u-boot";
+		reg = <0x000000 0x050000>;
+		read-only;
+	};
+
+	partition@1 {
+		label = "u-boot-env";
+		reg = <0x050000 0x020000>;
+		read-only;
+	};
+
+	partition@2 {
+		label = "firmware";
+		reg = <0x070000 0x780000>;
+	};
+
+	art: partition@3 {
```

Comment:
it seems that it was wrong before, but the unit-address (the number after the "@") should always match the address cell(s)
thus the correct value would be 7f0000 in this case (or in other words: use partition@7f0000 instead of partition@3)

the same comment also applies to all other partitions except the u-boot one

Reply:
Based on DTS specs, you are right. I fixed that.

(Likely it is mostly cosmetics, though, as it has been this 0 1 2 3 all the time in the WNDR3800 DTS file, and it has worked)